odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewTags/OverviewTagsStyles.ts

```diff
@@ -1,19 +1,12 @@
-import { Theme } from '@mui/material';
+import { styled } from '@mui/material/styles';
 
-import { WithStyles } from '@mui/styles';
-import createStyles from '@mui/styles/createStyles';
+export const CaptionContainer = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  marginBottom: theme.spacing(2.25),
+}));
 
-export const styles = (theme: Theme) =>
-  createStyles({
-    container: {},
-    captionContainer: {
-      display: 'flex',
-      alignItems: 'center',
-      justifyContent: 'space-between',
-      marginBottom: theme.spacing(2.25),
-    },
-    caption: {},
-    tagsContainer: { margin: theme.spacing(0, -0.5, 0, -0.5) },
-  });
-
-export type StylesType = WithStyles<typeof styles>;
+export const TagsContainer = styled('div')(({ theme }) => ({
+  margin: theme.spacing(0, -0.5, 0, -0.5),
+}));
```

Comment:
This one won't be necessary when you will use Box component in OverviewTags.tsx file in tags container.